skimage/feature/peak.py

```diff
@@ -85,9 +84,13 @@ def peak_local_max(image, min_distance=1, threshold_abs=None,
         the minimum intensity of the image.
     threshold_rel : float, optional
         Minimum intensity of peaks, calculated as `max(image) * threshold_rel`.
-    exclude_border : int or bool, optional
-        If nonzero int, `exclude_border` excludes peaks from
-        within `exclude_border`-pixels of the border of the image.
+    exclude_border : tuple of ints or bool, optional
+        If tuple of ints, the length of the tuple must match the input array's
+        dimensionality.  Each element of the tuple will exclude peaks from
+        within `exclude_border`-pixels of the border of the image along that
+        dimension.
+        If nonzero int, `exclude_border` excludes peaks from within
```

Comment:
Could you please put the "nonzero int" case first (before "tuple of ints") since I would expect it to be more common?

Reply:
Updated.